### PR TITLE
Wip/beol biblio

### PR DESCRIPTION
**Pull request by SepidehAlassi**
I have added a new resource for Editions and would like to merge now this branch to develop can you please review it?

### CODE REVIEW

**Comment by benjamingeer** [comment]
This looks OK to me. Could you please also move `Person` and its properties out of `knora-base` and into `biblio-onto`?

**Comment by tobiasschweizer** [comment]
How would a biblio:Person relate to a beol:Person?

**Comment by benjamingeer** [comment]
I just mean that I don't think `Person` is ready to be in `knora-base` yet. It's still too BEOL-specific. What you do in project-specific ontologies is up to you, but please move it out of `knora-base`.

**Comment by SepidehAlassi** [comment]
then the biblio-onto will be specific to beol, and biblio:Person will be a beol:Person. Later on when we want to make bibliography ontology generic in order to be used by other projects, we should think about adding a Person to Knora-base though.

**Comment by benjamingeer** [comment]
I think it's really OK for the bibliography ontology to be specific to BEOL for now. Later, when there are other projects that need bibliographies, we can work on making it generic.

File: webapi/_test_data/ontologies/biblio-onto.ttl [hunk 1]
```diff
@@ -9,6 +9,7 @@
 @prefix event: <http://purl.org/NET/c4dm/event.owl#> .
 @prefix knora-base: <http://www.knora.org/ontology/knora-base#> .
 @prefix salsah-gui: <http://www.knora.org/ontology/salsah-gui#> .
+@prefix beol: <http://www.knora.org/ontology/beol#> .
 @base <http://www.knora.org/ontology/biblio-onto> .
```
Comment (tobiasschweizer) [inline webapi/_test_data/ontologies/biblio-onto.ttl]: Please check the prefixes for the biblio ontology. I think it should just say <http://www.knora.org/ontology/biblio> (without "onto").

Comment (tobiasschweizer) [inline webapi/_test_data/ontologies/biblio-onto.ttl]: You can compare it with "incunabula-onto.ttl"

**Comment by tobiasschweizer** [comment]
you can delete the branch once it was successfully merged

**Comment by subotic** [comment]
@SepidehAlassi you probably used 'Merge and Rebase' and not the 'Squash and Merge' button. We prefer to squash the commits, which keeps the history clearer. Also, I don't know why the 'Merge and Rebase' option was activated. I have deactivated it now, so this doesn't happen again.

**Comment by SepidehAlassi** [comment]
@subotic I merged using sourceTree, not with github, have never seen Squash and Merge option there. BTW , do we have any developer guideline about handling branches and such?

**Comment by subotic** [comment]
No, nothing written down. Then please just use Github in the future, and then you will be fine.

**Pull request merged**